backends/npu/kernels/funcs/npu_funcs.h

```diff
@@ -86,6 +86,9 @@ inline void TensorCopy(const Context& dev_ctx,
   C_Stream stream = static_cast<C_Stream>(dev_ctx.stream());
 
   auto size = src.numel() * paddle::experimental::SizeOf(src.dtype());
+  if (UNLIKELY(size) == 0) {
+    return;
+  }
 
```

Comment:
Add the judgment to input tensor size. Copying a zero tensor will raise the error in function AsyncMemCpyH2D